src/java.base/share/classes/java/lang/ThreadLocal.java

```diff
@@ -378,6 +417,12 @@ private static int prevIndex(int i, int len) {
             return ((i - 1 >= 0) ? i - 1 : len - 1);
         }
 
+        /**
+         * Construct a new map without a table.
+         */
+        ThreadLocalMap() {
```

Comment:
It might be possible for this to be `private`:
```suggestion
        private ThreadLocalMap() {
```

Reply:
Yes, this can be private.